packages/sdk/src/fetcher/serializer/index.ts

```diff
@@ -0,0 +1,15 @@
+import { KPIToken } from "../../entities/kpi-token";
+
+export type KPITokensObjectProp = { [address: string]: KPIToken };
```

Comment:
I think we can rename this to `KPITokensObjectProp` since it's not used as a prop for a component,
Also, we could use the `Record` utility type here, something like:

```
export type KPITokensObject = Record<string, KPIToken>;
```

Wdyt?

Reply:
I think it's a good suggestion, will update. Thanks!